docs/tasks/administer-cluster/cloud-controller-manager-daemonset-example.yaml

```diff
@@ -0,0 +1,63 @@
+
+# This is an example of how to setup cloud-controller-maanger as a Daemonset in your cluster.
+# It assumes that your masters can run pods and has the role node-role.kubernetes.io/master
+# Note that this Daemonset is unlikely to work straight out of the box for your cloud, this is
+# meant to be a guideline.
+
+---
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: cloud-controller-manager
+  namespace: kube-system
+---
+kind: ClusterRoleBinding
+apiVersion: rbac.authorization.k8s.io/v1beta1
+metadata:
+  name: system:cloud-controller-manager
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: cluster-admin
```

Comment:
Add BIG WARNING that this is DISCOURAGED or just link to a `system:cloud-controller-manager` ClusterRole and bind to that, and say to the user: fill in necessary verbs/resources here

Reply:
I think I'm going to add an actual ClusterRole in this example, but I need to hash out what verbs/resources it actually needs access to. I'm going to do some testing and find out, mind if I updated this in another PR?

Reply:
what API calls are cloud controllers responsible for making?

Reply:
IIRC:
* list/watch/update/delete nodes 
* list/watch/update services
* list/watch/update PVL

^ there could be more, I want to test it to be sure

Reply:
I haven't tested it but I think this is what the CCM needs currently. (was planning to next week)
```
- apiGroups:
  - ""
  resources:
  - nodes
  verbs:
  - get
  - list
  - watch
  - update
  - patch
  - delete
- apiGroups:
  - ""
  resources:
  - services
  verbs:
  - list
  - watch
  - patch
- apiGroups:
  - ""
  resources:
  - persistentvolumes
  verbs:
  - list
  - watch
  - patch
```

Reply:
Nice! This will be a good starting point for my tests too, thanks @jhorwit2!

Reply:
We could also use @liggitt's awesome new project to generate it :) https://github.com/liggitt/audit2rbac